lib/install/data/data.go

```diff
@@ -84,6 +84,12 @@ type Data struct {
 
 	ScratchSize string
 	Rollback    bool
+
+	SyslogConfig SyslogConfig
```

Comment:
Just FYI, you can still refer to it by the same name if you embed it, i.e.: `if conf.Diagnostics.SyslogConfig != nil { ... }`